app/instance-initializers/key-responder.js

```diff
@@ -0,0 +1,51 @@
+import Ember from 'ember';
+
+export default {
+  name: 'ember-key-responder-instance',
+
+  initialize: function(instance) {
+
+    //TextField/TextArea are currently uninjectable, so we're going to hack our
+    //way in
+    Ember.TextSupport.reopen({
```

Comment:
this shouldn't be on an InstanceInitializer, it should only be run once, not per instance.,